docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
       # Always UTC+05:45. Set to catch timezone problems.
       - TZ=Asia/Kathmandu
       - FERRETDB_HANDLER=postgresql
-      - FERRETDB_POSTGRESQL_URL=postgres://user@postgres:5432/dance
+      - FERRETDB_POSTGRESQL_URL=postgres://username:password@postgres:5432/dance
```

Comment:
I changed the username to be consistent with FerretDB's Docker compose [file](https://github.com/FerretDB/FerretDB/blob/main/docker-compose.yml#L24).

Reply:
That change means that authentication is effectively disabled for all tests that use MongoDB URI without credentials.

We need to remove them there, not update them.

Reply:
In other words, that's not different from `trust`

Reply:
I see, when I do remove the credentials SCRAM authentication fails. Should we call `SetBypassBackendAuth()` before [here](https://github.com/FerretDB/FerretDB/blob/main/internal/handler/msg_saslstart.go#L81) to bypass backend authentication?

```
byron@byron-ThinkCentre-M910q:~/workdir/dance$ mongosh "mongodb://username:password@localhost:27017/?authMechanism=SCRAM-SHA-1"
Current Mongosh Log ID:	65d7722489a2ee64c03f12e8
Connecting to:		mongodb://<credentials>@localhost:27017/?authMechanism=SCRAM-SHA-1&directConnection=true&serverSelectionTimeoutMS=2000&appName=mongosh+1.10.6
MongoServerError: [msg_saslstart.go:184 handler.(*Handler).scramCredentialLookup] [collection.go:57 postgresql.(*collection).Query] [registry.go:279 metadata.(*Registry).DatabaseGetExisting] [registry.go:132 metadata.(*Registry).getPool] [pool.go:128 pool.(*Pool).Get] [opendb.go:95 pool.openDB] [opendb.go:110 pool.checkConnection] failed to connect to `host=postgres user=root database=dance`: failed SASL auth (FATAL: password authentication failed for user "root" (SQLSTATE 28P01))
```

Reply:
We should ensure that the problem is not in the dance configuration, maybe test manually, then report the problem

Reply:
I tested it manually and it fails so I created an issue, see FerretDB/FerretDB/issues/4115.